docs/post-mortems/0000-post-mortem-template.md

```diff
@@ -0,0 +1,109 @@
+## Summary
+<!--
+Describe the reason and the consequences of the event as short and concise as possible.
+
+Example:
+The backend of Oslo Nøkkelen were unavailable for 31 minutes due to an increase in traffic that happened after a
+proaktiv melding.
+-->
+
+## Timeline
+<!-- 
+Describe the relevant activites in a timeline format. Remember activities leading up to the event being triggered.
+
+2019-12-30
+    23:30 A proactive message got sendt out to all the citizens of Oslo
+2019-12-31
+    08:23 Oslonøkkelen had been downloaded 30.000 times, distributed evenly across iOS and Android, during the past 3 hours
+    09:09 DOWNTIME START - backend fails due to the increased load
+    09:14 Received Slack notifications due to an increase in 500 status codes in the backend
+    09:16 Initiated an investigation into the issue
+    09:18 EVENT START - Team creates the Slack channel #citykey-incident-backend to better cooperate on the issue
+    09:20 Manually scaling up the number of backend instances
+    09:24 Found the error. The backend fails when it tries to read an item in the database that doesn't exist. Only happens upon a users first login
+    09:28 Pull request with a fix created: http://github.com/oslokommune/something/something/pr/298
+    09:34 Pull request 298 merged and deployed to production
+    09:39 PROBLEM FIXED - Amount of 500 status code requests are decreasing
+    09:40 DOWNTIME END - Amount of 500 status codes are down to zero
+    10:10 EVENT END - Reaches criteria of 30 minutes with normal activity
+-->
+
+## Impact
+<!--
+Describe the consequences this had for the organization.
+
+Example:
+- Approximately 30.000 users downloaded and experienced an error using the app.
+- Users already logged in experienced some minor delay in the app
+-->
+
+## Cause(s)
+<!--
+Describe relevant factors that played a part in causing the event.
+
+Example:
+The error happened due to a combination of:
+- A spike of new users
+- A bug in the backend code for user creation
+-->
+
+## Solution
+<!--
+Describe how the problem was fixed.
+
+Example:
+- Manually scaled up number of backend instances letting regular traffic stay unaffected
+- Created a fix for the database schema
+-->
+
+## What went well
+<!--
+Describe what went well trying to handle the event.
+
+Example:
+- Alerting mechanisms worked brilliantly when errors started comming in
+- Deploying the database update was fast
+-->
+
+## What could we have done better
+<!--
+Describe what went wrong trying to handle the event.
+
+Example:
+- Had to manually roll back the database while testing fixes
+-->
+
+## Where were we lucky
```

Comment:
I suggest removing this until we deem it necessary. Unless someone has any arguments for why it should stay?

I trust that everyone in this team will identify luck and make sure it happens by default instead with or without this point

Reply:
I suggest keeping it. We used it actively both in Dataplattform and Schibsted to note things that contributed to the result not being as bad but were outside of our control, such as timing of the incident (e.g. low usage hours).

Reply:
Did you get any value from noting down these things?

Reply:
I think it's useful to note things that worked in our favour, but that we don't necessarily have control over.

Your comment above "everyone in this team will identify luck and make sure it happens by default instead" indicates things that we can control (i.e. so we can make it happen by default)?

Reply:
I see that we have used that section in 4 out of 4 incident reports from Dataplattform on Confluence, so I would say it was useful. You can always leave it empty in your review if it's not relevant. If we see that we don't have a use for the secton, we can always remove it from the template later?

Reply:
Definitely. I'm just used to adding things when we need them, instead of adding things first, then removing them later.

Trying to mitigate wasting time writing things that doesn't get used. Anyways, both approaches work

Reply:
You asked for our $.02. History has shown me that the section is useful 🤷

Reply:
Yeah! And I appriciate it. Just want to test things to make sure we're not doing things "just because we've always done it like this"

Reply:
The incident report is in big part a record of what happened. Being aware of conditions that can work in our favour (or against us) is useful in itself, is it not? Regardless of whether or not it is something we can influence/control and thus take action on.

Reply:
I looked through the incident reports from Oslonøkkelen and Skjemautvikling on Confluence as well, and the majority (but not all) of them also included items in the "We got lucky" seciton.

Reply:
Most of the things that end up in this section don't really belong in "What went well", e.g. "We don't have many active users on the service yet so noone was affected" is not really something that "went well" 😁

Reply:
> The incident report is in big part a record of what happened. Being aware of conditions that can work in our favour (or against us) is useful in itself, is it not?

Might be, I just don't see how. 

> I looked through the incident reports from Oslonøkkelen and Skjemautvikling on Confluence as well, and the majority (but not all) of them also included items in the "We got lucky" seciton.

Included does not equal value. If someone at some point looked it up to answer some kind of question, then I'll agree it has value.

We obviously disagree here. We'll see what the other's chime in post chrismas. At least now they have some pros and cons to work with ^^

Reply:
It has value if it was useful in that incident review itself, to e.g. reason about impact? Otherwise don't include it of course. It cannot be a requirement that it has future value for someone going back to look at the review later?

Reply:
I don't have a strong opinion on this, but if a post mortem doesn't have anything to say on this point, the author can just remove it. Also, if we never use it, we can remove it from the template.

Reply:
Leaving it